### ISSUE TITLE
Oled theme

### ISSUE THREAD

**Issue by Toiletmason** [breezy-weather/breezy-weather#147]
### Describe your suggested feature

The black theme would be black.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to **[latest version](https://github.com/breezy-weather/breezy-weather/releases/latest)**.

**Comment by Reed-Schimmel** [comment]
I would also love this feature

**Comment by Aturex1** [comment]
Me too, but I'm happy enough that there is a dark theme already

**Comment by papjul** [comment]
Why isn't the current dark theme enough?

**Comment by Aturex1** [comment]
It is (at least for me) but it would be a nice extra to have a OLED-Theme for OLED-Smartphones

**Comment by Korb** [comment]
The requested feature is sometimes called "pure black background".

**Comment by papjul** [comment]
I'm opening this to contributions. Should be implemented as a new option of Dark mode preference:
- Follow system
- Light
- Dark
- Pure black

**Comment by papjul** [comment]
"Follow system" follows preference of the system which only offers light and dark to my knowledge. If there is a system preference for Pure black in standard Android that I missed, we can indeed automatically use it for the "Follow system" preference. Let me know if you find anything?

**Comment by papjul** [comment]
Yes, I know, but if I mentioned that I wanted to make it that way because I always found it was not a good thing:
- It creates the ability to choose a light theme with a "Pure black" toggle (or system preference which can be light, with "Pure black"), which doesn't make sense. We could play with disabled option for the light theme, but not with "follow system"
- It creates an isolated new option, which I would prefer avoiding; we already have a toggle for day/night mode
- If system preference is dark, but Breezy Weather applies pure black, technically, it doesn't follow system preference

I get what you're trying to achieve, but you all seemed to prefer OLED to reduce eye strain, so I find it curious that you now want both light and pure black at the same time?

Note that if you set Dark mode to Pure black and Day/Night mode enabled, you can still get an automatic light/pure black.

I just want to avoid adding too many options for edge-cases. I know it's just one, but the more we add, the more users looking for the most common options can't find it. I regularly get user reports asking where they can set units…
